### PR TITLE
Extract migrations as CRDT seeds and persist `db/seeds.dump`

### PR DESCRIPTION
**Pull request by puffo**
- refactors all migrations containing platform/factory/metafactory information into `artistIntegrations.ts`
- loads data into DB via CRDT seeds in migration 38
- changes the `value` on `crdt_state` from varchar(255) to varchar(1000) as some bigger typeMetadata instructions needed it
- adds yarn commands for saving, clearing, and restoring seeds table

### CODE REVIEW

**Comment by musnit** [comment]
Liking good. Do u think it would also make sense to:
 - save seeds table in backup
 - remove migrations for seeds entirely and just insert the seeds via our Postgres backup tables with something like yarn restore-seeds
 - do rebuild with clean migration state

?

I guess only thing may b that we may wanna use plaintext pg_dump backups for the seeds table instead of compressed ones so that they’re easier to read and have nicer diffs on GitHub when we save new versions

**Comment by puffo** [comment]
Yeah, plaintext seeds table dump would make a lot of sense. 

The existing compressed DB dumps might get out of sync with the seeds table dump but it probably isn't a huge problem given that we can always do a full rebuild and come to the same end state eventually.

**Comment by puffo** [comment]
> remove migrations for seeds entirely

Yes to removing and renumbering the migrations, but I do think we can use a handful (if not all) of the `artistMigrations.ts` to define some kind of acceptable API docs for the seeds API's payloads. 

So I'd like to keep those in for a little while longer. Otherwise we should at least keep track of a commit for us to reference later on

**Comment by musnit** [comment]
> The existing compressed DB dumps might get out of sync with the seeds table dump but it probably isn't a huge problem given that we can always do a full rebuild and come to the same end state eventually.

yeh sounds ok

File: src/db/migrations/37-change-crdt-state-table-value-column-capacity.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { Knex } from 'knex';
+
+import { Table } from '../db';
+import { tableNameToViewName, updateViews } from '../migration-helpers';
+
+export const up = async (knex: Knex) => {
+  await knex.schema.dropView(tableNameToViewName(Table.crdtState));
+  await knex.schema.alterTable(Table.crdtState, table => {
+    table.string('value', 1000).alter();
```
Comment (musnit) [inline src/db/migrations/37-change-crdt-state-table-value-column-capacity.ts:9]: oh nice catch. should this maybe be even longer/bigger? it looks like the biggest we have in practice that could potentially be exposed is 20000, so i think going with that should be fine

**Review by musnit — APPROVED** [review]
one small comment, but otherwise looks good, thanks!

do you think we should do a full rebuild? if so, then maybe we can just delete all the migrations instead of deprecating them?

**Pull request merged**